src/main/webapp/transcript.js

```diff
@@ -69,3 +74,11 @@ function appendPTagToContainer(text, container) {
   container.appendChild(pTag);
   return pTag;
```

Comment:
nit: maybe have a different name other than pTag

Reply:
just for reference, the html `p` tag defines a paragraph of text.  So maybe you could call it `appendParagraphToContainer`?

Reply:
Updated to `appendParagraphToContainer` Thanks!